Python/Docscanner_Python/README.md

```diff
@@ -0,0 +1,16 @@
+# Python Docscanner
+
+It takes the image of your notes as input and give out black and white readable processed image as an output.
+## Setup instructions
+
+Make sure you have installed numpy and opencv-python using pip
+
+## Detailed explanation of script, if needed
+
+Enter your image file name as input, and opencv will apply adaptive thresholding and binary thresholding. Output image will be named as 'output.jpg'. 
+
+## Author(s)
+
+Name : Dibyajyoti Mondal  
+Github username : djm-1
```

Comment:
Add a hyper-link to your name with your Github Profile, if you want to be credited with your Script